dev-aws/kafka-shared-msk/account-identity/verification.tf

```diff
@@ -33,3 +33,11 @@ resource "kafka_topic" "account_identity_verification" {
   partitions         = 3
   replication_factor = 3
 }
+
+# Existing iam dpd mapper config is in ../iam/iam.tf
+module "iam_dpd_mapper" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = [(kafka_topic.account_identity_verification.name)]
+  consume_groups   = ["iam.dpd-mapper"]
```

Comment:
If this consume groups is already defined in the other iam_dpd_mapper, it doesn't need to be here too. It will create the same ACL in kafka that is already defined in iam.

Reply:
gotcha, makes sense 👍